### PR TITLE
Added Ore Processing to OpenComputers.

### PR DESCRIPTION
**Pull request by Vexatos**
As requested by https://twitter.com/SangarWasTaken/status/585822670899568641.

Yes, this is a totally serious Pull Request.

### CODE REVIEW

**Comment by magik6k** [comment]
WAT

**Comment by asiekierka** [comment]
@magik6k - Context: https://twitter.com/asiekierka/status/585807276260352000

File: assets/misc/oreprocessing.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+This is ore processing for OpenComputers.
+You might say, 'Hey, but this is just a text file, how would it do ore processing?'
+And you would be right, because this is indeed just a plain text file, and it
+won't process your ores for you. What do you expect? Something to do it automatically?
+In OpenComputers? No, this is how you do ore processing in OpenComputers:
+Step 1: Start Minecraft
+Step 2: Get a furnace
+Step 3: Smelt your ores in it
```
Comment (fnuecke) [inline assets/misc/oreprocessing.txt]: Plzfix, should say "have a robot smelt ores in it".

Comment (Vexatos) [inline assets/misc/oreprocessing.txt]: Sorry, will fix.

**Comment by Vexatos** [comment]
Fixed.

**Comment by fnuecke** [comment]
Thanks! And with this, OC wins the race. [insert manical laughter]

**Comment by Vexatos** [comment]
:laughing: :smiling_imp:

**Comment by evieluvsrainbows** [comment]
Uh, just wondering, but what the heck is the point of this? I find it kinda silly. :stuck_out_tongue:

**Pull request merged**